controllers/SiteController.php

```diff
@@ -308,26 +294,103 @@ public function actionChangeLanguage($lang)
         }
     }
 
-    public function actionDesignList()
-    {
-        return $this->render('design-list');
+    public function actionDesignList($viewMode = NULL) {
+        if (!empty($viewMode)) {
+            $viewMode = 1;
+        } else {
+            $viewMode = 0;
+        }
+        $query = new Query;
+        $query->select(['moqup.*', 'user.username as username'])
+                ->from('moqup')
+                ->where(['!=', 'user_id', Yii::$app->user->id])
+                ->leftJoin('user', 'moqup.user_id = user.id')
+                ->all();
+
+        $command = $query->createCommand();
+        $moqups = $command->queryAll();
+
+        $your_moqups_qry = new Query;
+        $your_moqups_qry->select(['moqup.*', 'user.username as username'])
+                ->from('moqup')
+                ->where(['user_id' => Yii::$app->user->id])
+                ->leftJoin('user', 'moqup.user_id = user.id')
+                ->all();
+
+        $your_moqups_cmd = $your_moqups_qry->createCommand();
+        $your_moqups = $your_moqups_cmd->queryAll();
+
+        return $this->render('design-list', ['viewMode' => $viewMode, 'moqups' => $moqups, 'your_moqups' => $your_moqups]);
     }
 
-    public function actionDesignView()
-    {
-        return $this->render('design-view');
+    public function actionDesignView($id) {
+        $moqup = Moqup::find()
+                ->where(['id' => $id])
+                ->one();
+        $css = Css::find()
+                ->where(['moqup_id' => $id])
+                ->one();
+        return $this->render('design-view', ['moqup' => $moqup, 'css' => $css]);
     }
 
-    public function actionDesignEdit()
-    {
-        return $this->render('design-edit');
+    public function actionDesignAdd() {
+        if (Yii::$app->request->isPost) {
+            $formatter = \Yii::$app->formatter;
+            $now = $formatter->asDateTime('now');
+            $now = strtotime($now);
+
+            $moqup = new Moqup;
+            $moqup->user_id = Yii::$app->user->id;
+            $moqup->title = Yii::$app->request->post('title');
+            $moqup->html = Yii::$app->request->post('html');
+            $moqup->created_at = $now;
```

Comment:
@sapkaljagruti You may use timestamp behavior which takes care for this. [Reference.](https://www.yiiframework.com/doc/api/2.0/yii-behaviors-timestampbehavior)